### ISSUE TITLE
Misc. CSS fixes

### ISSUE THREAD

**Issue by cnk** [hackforla/website#759]
### Dependency
#577 - has been closed/completed

- [ ] Move .meetings-message css class to an appropriate shared file and remove duplicates
- [ ] active::after is used in 2FA guide page and getting-started-page.html/scss - move somewhere and remove dups

- [  ] Fix another issue with active:after. Because active::after is defined in getting-started-page.scss for desktop views, it applies to events page as well. Which means, if you click the "Meeting Events" header under "Our Events", it will add a drop down arrow to the left of it. Click it again and it makes the content disappear. 

(These issues were noted by @KianBadie in PR #758 )

**Comment by alexandrastubbs** [comment]
@daniellen00 any update on this?

**Comment by daniellex0** [comment]
Sorry about the delayed response- still figuring this out now that I have joined the development environment.

**Comment by pawan92** [comment]
@daniellen00 Could you please update:
Progress:
Blockers:
Availability:
ETA:

**Comment by daniellex0** [comment]
Progress: I will get to this after I create shared scss files for reusable page cards and cards- components of this detail will be a little later in the process
Blockers: The other bigger reusable components in the design system need to be done before this

**Comment by pawan92** [comment]
@daniellen00 Any new updates:
Progress:
Blockers:
Availability:
ETA:

**Comment by ExperimentsInHonesty** [comment]
@kristine-eudey will talk to @daniellex0  about what to do with this issue, 
Is this a front end issue now?  
Does this still need to be done?

**Comment by kristine-eudey** [comment]
I talked with @daniellex0 who told me this should now be a front-end issue, and the developers should determine if they think these items still need to be done.

**Comment by Aveline-art** [comment]
This issue does not need work since the action items had been either fixed or in progress.
